Bottom: db1c09bf1ee5cf0b3d74ced0894853e55d09bfe2
Top:    5c7d80dbd0d76712225e4a849e09ed90273284f1
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-03 14:56:32 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE-for-autofs-mounts.patch

---

diff --git a/lib/mounts.c b/lib/mounts.c
index 7a00f23..494fc7b 100644
--- a/lib/mounts.c
+++ b/lib/mounts.c
@@ -104,14 +104,11 @@ unsigned int query_kproto_ver(void)
 		return 0;
 	}
 
-	if (mount("automount", t_dir, "autofs", mountflags, options)) {
-		mountflags &= ~MS_UNBINDABLE;
-		if (mount("automount", t_dir, "autofs", mountflags, options)) {
-			close(pipefd[0]);
-			close(pipefd[1]);
-			rmdir(t_dir);
-			return 0;
-		}
+	if (mount("automount", t_dir, "autofs", MS_MGC_VAL, options)) {
+		close(pipefd[0]);
+		close(pipefd[1]);
+		rmdir(t_dir);
+		return 0;
 	}
 
 	close(pipefd[1]);
